tools/device-listing/Program.cs

```diff
@@ -52,6 +52,7 @@
     "power",
     "voc",
     "thermocouple",
+    "weight",
```

Comment:
while you are there, don't you want to make the little changes used here to move to a json file? :-) https://github.com/nanoframework/Samples/tree/main/device-listing

If not, no worry, that can be done later.

Reply:
I'd prefer to do that later, unfortunately don't have much time this week as I'm preparing for .NET Conf and I'd like the new devices to be available on device listing since they're used in the presentation :-)